### PR TITLE
UI implementation

### PR DESCRIPTION
**Pull request by MrBrN197**
- ✔ Created a `main.rb` file that will serve as your console app entry-point
- ✔ Created functionality for the user interface that will:
  - ☑ List all books.
  - ☑ List all people.
  - ☑ Create a person (teacher or student, not a plain `Person`).
  - ☑ Create a book.
  - ☑ Create a rental.
  - ☑ List all rentals for a given person id.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './rental'
+
+class App
+  def initialize
+    puts teacher.id
+    @books = []
```
Comment (Mihndim2020) [inline main.rb:10]: - [ ] Please your app is not running from my end, checkout the error message the image below and kindly fix the issue.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './rental'
+
+class App
+  def initialize
+    puts teacher.id
```
Comment (Mihndim2020) [inline main.rb]: - [x] Please your app is not running from my end, checkout the error message the image below and kindly fix the issue.
teacher

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Good job with Gitflow

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

***I will leave the comment here since I do not have access to your `readme` file for this PR***

Please also consider:
- [ ] Please kindly add instructions in your `readme` file to pull and run your project locally. This will help non technical users to quickly get an idea of how your project works and some are potential recruiters.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @MrBrN197 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

**Pull request merged**